lms/templates/api_admin/terms_of_service.html

```diff
@@ -0,0 +1,104 @@
+## mako
+<%page expression_filter="h"/>
+<%inherit file="../main.html"/>
+<%
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+%>
+
+<div id="api-access-wrapper">
+  <h1 id="api-access-request-header">${_("Terms of Service for {platform_name} APIs").format(platform_name=settings.PLATFORM_NAME)}</h1>
+  <h2 class="api-access-request-subheading">${_("Effective Date: April 7th, 2016")}</h2>
+
+  <p class="api-tos-body">${_("Welcome to {platform_name}. Thank you for using {platform_name}'s Course Discovery API and any additional APIs that we may offer from time to time (collectively, the \"APIs\"). Please read these Terms of Service prior to accessing or using the APIs. These Terms of Service, any additional terms within accompanying API documentation, and any applicable policies and guidelines that {platform_name} makes available and/or updates from time to time are agreements (collectively, the \"Terms\") between you and {platform_name}. By accessing or using the APIs, you accept and agree to be legally bound by the Terms, whether or not you are a registered user. If you do not understand or do not wish to be bound by the Terms, you should not use the APIs.").format(platform_name=settings.PLATFORM_NAME)}</p>
```

Comment:
I'm doing some layout in addition to styling, so it makes sense to have a way to target just the body elements. I could target `p` instead of `.api-tos-body`, but class selectors seem neater than element selectors.